src/api_server/peer.cpp

```diff
@@ -0,0 +1,163 @@
+/**
+ * The Forgotten Server - a free and open-source MMORPG server emulator
+ * Copyright (C) 2016  Mark Samman <mark.samman@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "../otpch.h"
+
+#include "peer.h"
+#include "server.h"
+#include "router.h"
+#include "responder.h"
+#include "../tasks.h"
+
+namespace http
+{
+
+using Minutes = boost::posix_time::minutes;
```

Comment:
deadline_timer only supports posix_time. I tried using steady timer instead, but i had issues compiling it with clang.